python/lsst/ctrl/mpexec/cli/cmd/commands.py

```diff
@@ -236,6 +281,8 @@ def pre_exec_init_qbb(repo: str, qgraph: str, **kwargs: Any) -> None:
 @processes_option()
 @ctrlMpExecOpts.pdb_option()
 @ctrlMpExecOpts.profile_option()
+@ctrlMpExecOpts.coverage_option()
+@ctrlMpExecOpts.coverage_packages_option()
```

Comment:
The options have been added here but the code to turn on coverage is missing from the function and needs to be added.